profile/README.md

```diff
@@ -15,16 +15,36 @@
 
```

Comment:
ReactApp Env variables for traverse ui should be included:

REACT_APP_BACKEND_URL=http://localhost:8000/
REACT_APP_STORAGE_SERVICE_URL=http://localhost:8080

Reply:
Added